torch/fx/experimental/proxy_tensor.py

```diff
@@ -82,29 +92,35 @@ def unwrap_proxy(e):
         args[0].proxy = proxy_out
         proxy_out.node.meta['tensor_meta'] = _extract_tensor_metadata(args[0])
 
-    with no_dispatch():
-        real_out = func_overload(*args, **kwargs)
-
+    real_out = func_overload(*pytree.tree_map(unwrap_elem, args), **pytree.tree_map(unwrap_elem, kwargs))
+    # Seems like func isn't marked as inplace_view for some reason?
```

Comment:
overload packets don't have tags as it is not clear how overload tags should be merged.

Reply:
sorry, I meant, seems like "func_overload" isn't marked as inplace_view for `resize_`.

Reply:
Should this tag be added to functions that are maybe inplace_view but not always?

Reply:
eh... probably? Maybe we need another tag lol